e2e/e2e-prow.sh

```diff
@@ -1,39 +0,0 @@
-#!/bin/bash
```

Comment:
This is specific to GKE testsuites, I think we still need this setup script, even if the runner goes away.

Reply:
Ah I didn't notice the apt-get calls. I think we want these to be assumed by the framework. Installing components on demand in a test framework is scary-ish - not to mention that installation probably depends on the user's OS, e.g. if the user is running from Mac OSX this won't work. We should have separate scripts that aren't invoked in the build lifecycle for these.

Reply:
Yes, did not intend for these to be part of the testsuite. These are external "setup" steps and the entrypoint for the k8s CI system.